charts/incubator/strongswan/values.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: ghcr.io/reitermarkus/strongswan
+  # -- image tag
+  tag: latest
+  # -- image pull policy
+  pullPolicy: Always
+
+# -- environment variables. See more environment variables in the [strongswan documentation](https://github.com/reitermarkus/strongswan).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # VPN_NAME: Example VPN # VPN display name.
+  # WIFI_SSID: Example WiFi # Disable VPN when connected to this WiFi network.
+  # VPN_DOMAIN: vpn.example.org # FQDN for accessing the VPN.
+  # VPN_P12_PASSWORD:
+  #   valueFrom:
+  #     secretKeyRef:
+  #       name: strongswan-credentials
+  #       key: p12-password
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        enabled: false
+        primary: false
+      isakmp:
+        enabled: true
+        port: 500
+        targetPort: 500
+        protocol: UDP
+      ike:
+        enabled: true
+        port: 4500
+        targetPort: 4500
+        protocol: UDP
+
+# -- Security contexts required for container.
+# @default -- See values.yaml
+securityContext:
+  capabilities:
+    add:
+      - NET_ADMIN
+
+hostNetwork: true
+dnsPolicy: ClusterFirstWithHostNet
+
+# -- Configures probes for the chart. Normally this does not need to be modified.
+# @default -- Disabled
+probes:
+  liveness:
+    enabled: false
+  readiness:
+    enabled: false
+  startup:
+    enabled: false
```

Comment:
Should the probes always be disabled or should it be possible for the user to enable them?

Reply:
There is no HTTP service to probe, so it should always be disabled. I'm not sure if there is another way to probe StrongSwan.

Reply:
No problem... In that case, something like this https://github.com/k8s-at-home/charts/blob/master/charts/stable/modem-stats/templates/common.yaml#L22L29 could be even better, since it prevents users from "accidentally" enabling the probes somehow.

Reply:
Okay, I found a simple command which should work as a rudimentary probe.